src/styles/card.css

```diff
@@ -0,0 +1,122 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  width: 200px;
+
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 3px;
+
+  display: flex;
+  align-content: center;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
+.card__content {
+  display: flex;
+  align-content: center;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: stretch;
+
+  width: 170px;
+}
+
+.card__prodact-img {
+  margin-top: 32px;
+  height: 134px;
+}
+
+.card__monitor-title {
+  margin-top: 40px;
+
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
+
+  display: flex;
+}
+
+.card__monitor-code {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+
+  color: #616070;
+
+  display: block;
+
+  margin-top: 4px;
+}
+
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+
+  margin-top: 16px;
+}
+
+.card__reviews-text {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+
+  display: flex;
+  align-content: flex-end;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+
+  margin-top: 24px;
+}
+
+.card__price-text {
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__price-value {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__button {
+  height: 40px;
+  background-color: #00acdc;
+  border-radius: 5px;
+  border: 1px solid #00acdc;
+
+  color: #fff;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 14px;
+  font-weight: 700;
+
+  display: flex;
+  align-content: center;
+  justify-content: center;
+  flex-wrap: wrap;
+
+  margin-top: 16px;
+  margin-bottom: 16px;
+
+  transition: 0.3s;
```

Comment:
specify the properties here, that can be changeable
by default it will work for all properties and it can affect on website performance in bad way